### PR TITLE
Another batch of Hungarian translations

### PR DESCRIPTION
**Pull request by balagan73**

### CODE REVIEW

File: config/translations/hu/migrate.debug.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
 arguments:
-    tag: 'Migrate tag'
+    tag: 'Költöztetési címke'
 messages:
-    id: 'Migráció Id'
+    id: 'Költöztetési azonosító'
     description: Leírás
-    no-migrations: 'There aren''t migrations available try to execute command: migrate:setup:migrations'
-    tags: Tags
+    no-migrations: "Nincsenek rendelkezésre álló költöztetések, meg kell próbálni a  „migrate:setup:migrations” parancs végrehajtását"
```
Comment (enzolutions) [inline config/translations/hu/migrate.debug.yml]: Opening double quotes are wrong

File: config/translations/hu/migrate.execute.yml [hunk 2]
```diff
@@ -1,38 +1,38 @@
...
+    db-host: 'Adatbázis-gazdagép'
+    db-name: 'Adatbázis neve'
+    db-user: 'Adatbázis-felhasználó'
     db-pass: 'Adatbázis jelszó'
-    db-prefix: 'Adatbázis Prefix'
-    db-port: 'Adatbázis Port'
-    invalid-migration-id: '"%s" érvénytelen migráció id'
+    db-prefix: 'Adatbázis előtag'
+    db-port: 'Adatbázis port'
+    invalid-migration-id: 'Érvénytelen „%s” költöztetési azonosító'
```
Comment (enzolutions) [inline config/translations/hu/migrate.execute.yml]: Opening quotes are wrong

**Comment by enzolutions** [comment]
@balagan73 there are also some conflict, please fix and push changes to this PR. thanks for your help

**Comment by balagan73** [comment]
Merge conflicts are solved, and missing quotes are added back.

**Comment by enzolutions** [comment]
@balagan73 changes look ok, bu, in execution I got the following error
```
commands.test.run.yml A YAML file cannot contain tabs as indentation at line 21 (near "	").
```

You need to setup your environmenet to test your own files first, ping me if you have problems doing that.

**Comment by balagan73** [comment]
The test didn't say which yaml file had the problem?

**Comment by enzolutions** [comment]
Thanks @balagan73

**Pull request merged**